OpenRA.Mods.Common/UpdateRules/Rules/AddNukeLaunchAnimation.cs

```diff
@@ -0,0 +1,42 @@
+﻿#region Copyright & License Information
+/*
+ * Copyright 2007-2018 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System.Collections.Generic;
+
+namespace OpenRA.Mods.Common.UpdateRules.Rules
+{
+	public class AddNukeLaunchAnimation : UpdateRule
+	{
+		public override string Name { get { return "Add 'WithNukeLaunchAnimation' and remove 'ActivationSequence'"; } }
```

Comment:
These should mention `NukePower.ActivationSequence` so users know what trait you are talking about.